app/controllers/courses_controller.rb

```diff
@@ -25,14 +25,9 @@ def apply
 
     Rails.logger.info("Course apply conversion. Provider: #{course.provider.provider_code}. Course: #{course.course_code}")
 
-    # If course provider is University of Bolton and Maths or Computing
-    if course.provider.provider_code == "8N5" && (course.course_code == "V595" || course.course_code == "Y710")
-      case course.course_code
-      when "V595"
-        redirect_to "https://evision.bolton.ac.uk/urd/sits.urd/run/siw_ipp_lgn.login?process=siw_ipp_app&code1=1101-D&code2=0001"
-      when "Y710"
-        redirect_to "https://evision.bolton.ac.uk/urd/sits.urd/run/siw_ipp_lgn.login?process=siw_ipp_app&code1=1100-D&code2=0001"
-      end
+    # If course provider is University of Bolton
+    if course.provider.provider_code == "8N5"
+      redirect_to "https://www.apply-for-teacher-training.service.gov.uk/candidate/account?providerCode=#{course.provider.provider_code}&courseCode=#{course.course_code}"
     else
       redirect_to "https://www.apply-for-teacher-training.service.gov.uk/candidate/apply?providerCode=#{course.provider.provider_code}&courseCode=#{course.course_code}"
     end
```

Comment:
shouldn't the `if` be removed? unless `account` vs `apply  that is the key
`

Reply:
Yea /apply hits a page which gives the candidate the option between applying on the Apply service or on UCAS.

We want to skip this page for all courses from Bolton as candidates cannot apply via UCAS this cycle.